hail/src/main/scala/is/hail/annotations/Region.scala

```diff
@@ -22,8 +31,7 @@ object Region {
 //    those operations have to know the RegionValue's Type to convert
 //    within-Region references to/from absolute addresses.
 
-final class Region private (empty: Boolean) extends NativeBase() {
-  def this() { this(false) }
+final class Region private (blockSize: Int, empty: Boolean) extends NativeBase() {
```

Comment:
It doesn't look like you're using `blockSize` in the constructor. Doesn't `nativeCtor` need to take a `blockSize`?

Reply:
oops, yes, fixed.